console/console-init/ui/src/modules/project-detail/components/AccessCredentials/AccessCredentials.tsx

```diff
@@ -108,7 +108,8 @@ const AccessCredentials: React.FunctionComponent<IAccessCredentialsProps> = ({
             value={url}
             isReadOnly={true}
             enableCopy={true}
-            ariaLabel={"input-messaging-url"}
+            id={"messaging-tenant-id-input"}
+            ariaLabel={"messaging-tenant-id"}
```

Comment:
we should avoid to use dash (-) in ariaLabel message.
It should be something  like ariaLabel={"messaging tenant input field"}

Reply:
Fixed